scripts/ci/gke.sh

```diff
@@ -68,23 +68,31 @@ create_cluster() {
     if is_OPENSHIFT_CI; then
         require_environment "JOB_NAME"
         require_environment "BUILD_ID"
-        tags="${tags},stackrox-ci-${JOB_NAME:0:50}"
-        tags="${tags/%-/x}"
-        labels="${labels},stackrox-ci-job=${JOB_NAME:0:63}"
-        labels="${labels/%-/x}"
-        labels="${labels},stackrox-ci-build-id=${BUILD_ID:0:63}"
-        labels="${labels/%-/x}"
+        build_num="${BUILD_ID}"
+        job_name="${JOB_NAME}"
     else
```

Comment:
do we need to is_GITHUB_ACTIONS clause?

Reply:
No. For scanner v2, we do all GKE cluster provisioning in OSCI (I'm actually not entirely sure why provisioning GKE clusters in GHA is supported in stackrox)